### PR TITLE
Problem: Thaler Testnet Guide has missing steps

### PR DESCRIPTION
**Pull request by calvinlauyh**
Solution: Add missing steps and tips for running on Azure

### CODE REVIEW

**Comment by lezzokafka** [comment]
Conflict with #168 : https://github.com/crypto-com/chain-docs/pull/168/files#diff-8858730c1ec093ea7a43196e843f91cc ?

File: docs/getting-started/thaler-testnet.md [hunk 1]
```diff
@@ -4,17 +4,30 @@ The Crypto.com Chain Testnet has been named as **“Thaler”**.
...
+You can look for your Intel CPU model on the [Intel Product Specifications website](https://ark.intel.com/content/www/us/en/ark.html#@Processors) and check if it has Intel® Software Guard Extensions (Intel® SGX) support.
+:::
 
-- Note that some motherboards may have only "software controlled" option where [an extra step is needed for enabling it](https://github.com/intel/linux-sgx/issues/354#issuecomment-447961815).
+- 4GB RAM
+- 100GB storage space
 
-- You may also need to install libzmq (e.g. [libzmq3-dev](https://packages.ubuntu.com/xenial/libzmq3-dev) package in Ubuntu 18.04).
+::: tip NOTE
+If you are using Microsoft Azure, you will need to create an instance of [Azure Confidential Computing](https://portal.azure.com/#create/microsoft-azure-compute.acc-virtual-machine-v2acc-virtual-machine-v2) with the above hardware requirement.
```
Comment (lezzokafka) [inline docs/getting-started/thaler-testnet.md]:
```suggestion
If you are using Microsoft Azure, you will need to create an instance of [Azure Confidential Computing (login required)](https://portal.azure.com/#create/microsoft-azure-compute.acc-virtual-machine-v2acc-virtual-machine-v2) with the above hardware requirement.
```

File: docs/getting-started/thaler-testnet.md [hunk 2]
```diff
@@ -23,6 +36,32 @@ has Intel PSW and other dependencies pre-installed
...
+If you are running on Azure Confidential Computing machines, you will only need to install the Intel SGX PSW 2.9 and aesmd service.
+:::
+
+- You can download the Linux SGX SDK installers from the Intel Open Source [website](https://01.org/intel-softwareguard-extensions/downloads/intel-sgx-linux-2.9.1-release). More details can be found in this [installation guide](https://download.01.org/intel-sgx/sgx-linux/2.9.1/docs/Intel_SGX_Installation_Guide_Linux_2.9.1_Open_Source.pdf).
+
+- Note that some motherboards may have only "software controlled" option where [an extra step is needed for enabling it](https://github.com/intel/linux-sgx/issues/354#issuecomment-447961815).
+
+### Step 0-1. Install aesmd service
+
+- After you have installed Intel SGX PSW 2.9 using the step 0-0 [installation guide](https://download.01.org/intel-sgx/sgx-linux/2.9.1/docs/Intel_SGX_Installation_Guide_Linux_2.9.1_Open_Source.pdf), run the following command to install aesm service:
```
Comment (lezzokafka) [inline docs/getting-started/thaler-testnet.md]:
```suggestion
- After you have installed Intel SGX PSW 2.9 using the [installation guide](https://download.01.org/intel-sgx/sgx-linux/2.9.1/docs/Intel_SGX_Installation_Guide_Linux_2.9.1_Open_Source.pdf) from [step 0-0](#step-0-0-install-intel-sgx-2-9), run the following command to install aesm service:
```

File: docs/getting-started/thaler-testnet.md [hunk 3]
```diff
@@ -319,7 +368,7 @@ stating who you are and your staking address (@devashishdxt or @lezzokafka would
 
 ### Step 3-b-3. Create a validator key pair
 
-- In a development mode, the full key pair is located in the `.tendetmint/config/priv_validator_key.json` ;
+- In a development mode, the full key pair is located in the `~/.tendetmint/config/priv_validator_key.json` ;
```
Comment (lezzokafka) [inline docs/getting-started/thaler-testnet.md]:
```suggestion
- In a development mode, the full key pair is located in the `~/.tendertmint/config/priv_validator_key.json` ;
```

**Review by lezzokafka — CHANGES_REQUESTED** [review]
Some quick suggestions, otherwise looks great!

**Review by lezzokafka — APPROVED** [review]

File: docs/getting-started/thaler-testnet.md [hunk 4]
```diff
@@ -4,17 +4,30 @@ The Crypto.com Chain Testnet has been named as **“Thaler”**.
...
 
-### Step 0. Install Intel SGX SDK 2.9 and other pre-requisites
+To run Crypto.com Chain nodes, you will need a machine with the following minimum requirements:
 
-- Make sure your CPU supports SGX, and it is enabled in BIOS. [This GitHub repository](https://github.com/ayeks/SGX-hardware) has more information about supported hardware and cloud vendors.
+- Intel SGX supported CPU
 
-- You can download the Linux SGX SDK installers from the Intel Open Source [website](https://01.org/intel-softwareguard-extensions/downloads/intel-sgx-linux-2.9.1-release). More details can be found in this [installation guide](https://download.01.org/intel-sgx/sgx-linux/2.9.1/docs/Intel_SGX_Installation_Guide_Linux_2.9.1_Open_Source.pdf).
+::: tip NOTE
+You can look for your Intel CPU model on the [Intel Product Specifications website](https://ark.intel.com/content/www/us/en/ark.html#@Processors) and check if it has Intel® Software Guard Extensions (Intel® SGX) support.
```
Comment (tomtau) [inline docs/getting-started/thaler-testnet.md:14]: looking at the CPU model isn't enough -- e.g. many MacBook CPU models are capable of executing SGX instructions, but there's no official way of enabling it.
I think that crowd-sourced repo https://github.com/ayeks/SGX-hardware  -- linked below https://github.com/crypto-com/chain-docs/pull/169/files#diff-8858730c1ec093ea7a43196e843f91ccR30 -- is probably the best resource so far

File: docs/getting-started/thaler-testnet.md [hunk 5]
```diff
@@ -4,17 +4,30 @@ The Crypto.com Chain Testnet has been named as **“Thaler”**.
...
-- You may also need to install libzmq (e.g. [libzmq3-dev](https://packages.ubuntu.com/xenial/libzmq3-dev) package in Ubuntu 18.04).
+::: tip NOTE
+If you are using Microsoft Azure, you will need to create an instance of [Azure Confidential Computing (login required)](https://portal.azure.com/#create/microsoft-azure-compute.acc-virtual-machine-v2acc-virtual-machine-v2) with the above hardware requirement.
+
+You will also need to create an extra disk in the Azure panel and mount it to the created instance. Instructions can be found [here](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/attach-disk-portal).
+:::
+
+## Common Setup
+
+### Step 0. Install Intel SGX 2.9 and other pre-requisites
```
Comment (tomtau) [inline docs/getting-started/thaler-testnet.md:28]: fortanix installation guide is also a good resource for common installation issues: https://edp.fortanix.com/docs/installation/help/

**Review by tomtau — APPROVED** [review]
looks ok

**Pull request merged**